simulator/docs/import-cluster-resources.md

```diff
@@ -1,16 +1,34 @@
 ### [Beta] Import your real cluster's resources
 
-The simulator can import resources from your cluster.
+There are two ways to import resources from your cluster. These methods can be used simultaneously.
+- Import resources from your cluster once when initializing the simulator.
+- Keep importing resources from your cluster.
 
-To use this, you need to follow these two steps
-- Set to `true` the `externalImportEnabled` value in the simulator server configuration.
-- Set the path of the kubeconfig file of the your cluster to `KubeConfig` value in the Simulator Server Configuration.
+#### Import resources once when initializing the simulator
+
+To use this, you need to follow these two steps in the simulator configuration:
+- Set `true` to `externalImportEnabled`.
+- Set the path of the kubeconfig file for the your cluster to `KubeConfig`. 
+  - This feature only requires the read permission for resources.
+
+```yaml
+externalImportEnabled: true
+kubeConfig: "/path/to/your-cluster-kubeconfig"
+```
+
+#### Keep importing resources
+
+To use this, you need to follow these two steps in the scheduler configuration:
+- Set `true` to `externalImportSynced`. 
+- Set the path of the kubeconfig file for the your cluster to `KubeConfig`. 
+  - This feature only requires the read permission for resources.
 
 ```yaml
-externalImportEnabled: false
+externalSchedulerSynced: true
 kubeConfig: "/path/to/your-cluster-kubeconfig"
 ```
 
-Then, the simulator imports resources from your cluster once when it's initialized.
+> [!NOTE]
+> When you enable `externalImportSynced`, deleting resources of the simulator cluster would cause a problem of syncing. Please avoid it except when you are debugging the simulator. When you delete a resource for debugging, make sure you reboot the simulator and the fake source cluster.
```

Comment:
```suggestion
> When you enable `resourceSyncEnabled`, deleting resources of the simulator cluster would cause a problem of syncing. Please avoid it except when you are debugging the simulator. When you delete a resource for debugging, make sure you reboot the simulator and the fake source cluster.
```